app/gui/src/dashboard/layouts/AssetsTable/EventListProvider.tsx

```diff
@@ -170,6 +171,7 @@ export function useAssetListEventListener(
     const events = initialEventsRef.current
     if (events && !alreadyRun) {
       // Event handlers are not idempotent and MUST NOT be handled twice.
+      // eslint-disable-next-line react-compiler/react-compiler
```

Comment:
Not sure we should suppress the react-compiler in that case.
The component is huge, and one line shouldn't lead to deopt for the component. IMO, better either leave the warning or refactor, e.g. fix or extract the deopt code into a separate hook/component